main.tf

```diff
@@ -107,12 +112,16 @@ module "runner-instance" {
     volumes    = ["/cache", "/certs/client"]
   }
 
+  runner_ami_filter = {
+    name = ["amzn2-ami-hvm-2.*-arm64-ebs"]
```

Comment:
I think this needs to be a variable, IIUC it must always correspond with `var.runner_instance_type` no?

Reply:
Yes. You are correct. Reverted and moved this to the infrastructure-live repo.